xform-marc21-to-xml/src/main/java/edu/stanford/AuthDBProperties.java

```diff
@@ -76,6 +80,37 @@ private void initDataSourceProperties(Properties properties) {
         this.userPass = properties.getProperty("PASS");
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
OK, I'll see if it can be done.  BTW, overriding these methods is common practice in Java, it's not something totally weird.  As it happens, doing so enables easier unit test code.  So you get just a couple methods on the object that save a ton of code in the specs.